ui/action-menu/src/ActionMenuItem.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from "react";
+
+import WPDS, { theme, styled } from "@washingtonpost/wpds-theme";
+
+import * as ActionMenuPrimitive from "@radix-ui/react-dropdown-menu";
+
+import { DropdownMenuItemProps as RadixDropdownMenuItemProps } from "@radix-ui/react-dropdown-menu";
+
+const NAME = "ActionMenuItem";
+
+export const ItemStyles = {
+  alignItems: "center",
+  background: theme.colors.secondary,
+  display: "flex",
+  flexBasis: "auto",
+  flexDirection: "row",
+  justifyContent: "flex-start",
+  flexWrap: "none",
+  transition: `background ${theme.transitions.fast} ${theme.transitions.inOut}`,
+  width: "100%",
+  "&:hover": {
+    backgroundColor: theme.colors.alpha25,
+    cursor: "pointer",
```

Comment:
<img width="221" alt="image" src="https://github.com/washingtonpost/wpds-ui-kit/assets/102534985/bb097f51-64fe-40cf-98df-630aac44fca1">

I'm fairly certain we don't want the focus ring to be visible when we hover

```suggestion
     "&:focus-visible": {
      outline: "none",
     },
```